addOns/pscanrules/CHANGELOG.md

```diff
@@ -4,7 +4,22 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
-
+### Changed
+- Replace usage of CWE-200 for the following rules (Issue 8712):
+    - Application Error Disclosure (Issue 8716)
```

Comment:
Do these issue references add anything?

Reply:
Good point, that the main one is referenced above is probably enough.